### PR TITLE
First push after creating Blacklight application (#22).

### PR DESCRIPTION
**Pull request by bwatson78**
All files were created by `rails generate blacklight:install --devise --marc --solr_version=latest' and `rake db:migrate`.

### CODE REVIEW

**Comment by bwatson78** [comment]
Screen Shot 2020-10-21 at 1 37 09 PM

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+* Ruby version 2.5.7p206
+
+* Rails version 5.1.7
+
+* Blacklight 7.4.1
+
+## Running locally
+
+1. Clone the git repo: `git@github.com:emory-libraries/blacklight-catalog.git`
```
Comment (rotated8) [inline README.md:13]:
```suggestion
1. Clone the git repo: `git clone git@github.com:emory-libraries/blacklight-catalog.git`
```

**Review by rotated8 — CHANGES_REQUESTED** [review]
One change to consider for the README, but otherwise good to go!

File: README.md (hunk 1 above)
Comment (bwatson78) [reply README.md]: That has been changed in the last commit.

**Review by rotated8 — APPROVED** [review]

**Comment by bwatson78** [comment]
In order to create a Blacklight 7.4.1 application, I had to go through the process below: 
1. `rails _5.1.7_ new .` from the terminal.
2. Add these to the `Gemfile`:
```
gem 'blacklight', '7.4.1'
gem 'solr_wrapper', '>= 0.3'
```
3. Also in the Gemfile, I had to restrict the ruby version explicitly, just like Lux:
```
ruby '>=2.5.0'
```

That helped me overcome errors that popped up with higher Rails and Ruby versions, such as extension of Concerns and random inclusion errors.

**Pull request merged**